src/chrome/content/rules/MichelbergerHotel.com.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+	Self-signed:
+		- mail
+		- vpn
+-->
```

Comment:
Wildcard DNS record

Reply:
@J0WI I'm not sure what to do here. I guess I should simply add a note in the comment and only include the domains that actually have content (not 404).

Reply:
`remote` sends the same response as the non-existent domains, so I removed it.

Reply:
What about the hosts in https://github.com/EFForg/https-everywhere/pull/17592#pullrequestreview-215209963?

Reply:
`shop`: no response at `:443`.
`store`: 404 on both http and https, can't find working url.

Reply:
Try with `http://shop.michelbergermonkey.com/germany/` or `http://store.michelbergermonkey.com/germany/`

Reply:
@J0WI Redirect to HTTP.

Reply:
Could you add them to the list?

Reply:
@J0WI Done.

Reply:
@J0WI Done.

Reply:
Thanks!